curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -428,79 +418,65 @@ Which of the following is NOT a valid string concatenation method in JavaScript?
 
 #### --text--
 
-What will be the output of the following code:
 
-```js
-let hello = "hello";
-if (true) {
-  let world = "world";
-}
-console.log(hello + " " + world);
-```
 
 #### --distractors--
 
-`hello world`
+
 
 ---
 
-`helloworld`
+
 
 ---
 
-`hello`
+
 
 #### --answer--
 
-`ReferenceError: world is not defined`
+
 
 ### --question--
 
 #### --text--
 
-What will be the output of the code below:
```

Comment:
Maybe we can have a question about camel casing here?

Reply:
like `Which of the following variable names uses camelCase correctly?`?

Reply:
Yeah